sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
 
 - Changed the return type of `StartCopy` API from a `Response<T>` to the particular `Operation<T>` type called `StartCopyShareFileOperation` directly.
 - String conversion functions of extensible enums were renamed from `Get()` to `ToString()`.
+- Renamed `GetSharePropertiesResult` to `ShareProperties`.
+- Renamed `GetShareStatisticsResult` to `ShareStatistics`.
+- Renamed `GetShareDirectoryPropertiesResult` to `ShareDirectoryProperties`.
+- Renamed `GetShareFilePropertiesResult` to `ShareFileProperties`.
+- Renamed `GetShareFileRangeListResult` to `ShareFileRangeInfo`.
```

Comment:
If the `Content-type` response header is not important, and you don't expect new headers to be added here, then keep it as `ShareFileRangeList`, don't use `Info`.

Otherwise, if you expect to add new response headers in the future, then drop the `Get`, keep the `Result`.

Reply:
@JeffreyRichter - what do you think in this case: `ShareFileRangeList` or `ShareFileRangeListResult`?

From a usability perspective, the first one is more natural, and my preference, and I would only go for the second if we have reason to believe new headers might be added in the future.

Reply:
For list operations, there is usually 1 header that is interesting (nextLink) and the body is a vector of some round-trippable resource type. So, you can leave off the Result suffix.

Reply:
I feel `ShareFileRangeList` implies this struct is iterable, but actually it's not.

I still suggest `GetShareFileRangeListResult`.

Reply:
I feel `ShareFileRangeList` implies this struct is iterable, but actually it's not.

I still suggest `GetShareFileRangeListResult`.